arbitrum-docs/node-running/running-an-archive-node.mdx

```diff
@@ -0,0 +1,69 @@
+# Running an Archive Node
+
+Note: If you’re interested in accessing the Arbitrum network but you don’t want to setup your own node, see our [Node Providers](./node-providers.mdx) to get RPC access to fully-managed nodes hosted by a third party provider.
+
+### Do You Need To Run An Archive Node?
+
+An archival node is a full (nitro) node that also builds an archive of historical states. Note that the use cases for having an archival node are rare (e.g., if you need to process historical data), and the great majority of users don't need it.
+
+- Followings specify the minimum hardware configuration required to setup an Arbitrum archival node:
+  - RAM: 16GB+ for Nitro and 32GB+ for Classic
+  - CPU: 4+ core CPU
+  - Storage: Minimum 2TB SSD (make sure it is extendable)
+  - Estimated Growth Rate: Around 128GB per week for Arbitrum One (Nitro)
+
+❗️Note: The minimum storage requirements will change over time as the Nitro chain grows. It is recommended to use more than the minimum requirements to run a robust archival node.
+
+### Required Artifacts
+
+- Latest Docker Image for Arbitrum One: `offchainlabs/nitro-node:v2.0.11-8e786ec`
+- Latest Docker Image for Arbitrum Classic: `offchainlabs/arb-node:v1.4.5-e97c1a4`
+
+- Arbitrum One Nitro Genesis Database Snapshot
+
+  - Use the parameter `--init.url="https://snapshot.arbitrum.io/mainnet/nitro.tar"` on first startup to initialize Nitro database
+  - sha256 checksum: `a609773c6103435b8a04d32c63f42bb5fa0dc8fc38a2acee4d2ab2d05880205c`
+  - size: 33.5573504 GB
+
+- Other chains do not have classic blocks, and do not require an initial genesis database
+
```

Comment:
We can add classic archive db url here: https://snapshot.arbitrum.io/mainnet/db.tar

Reply:
Do you happen to know what are its sha256 checksum and hash @Jason-Wanxt?